iphone/Classes/TiErrorController.m

```diff
@@ -16,7 +16,19 @@ @implementation TiErrorController
 - (id)initWithError:(NSString *)error_
 {
   if (self = [super init]) {
-    error = [error_ retain];
+    error = [NSString stringWithFormat:@"⚠️ %@", [error_ retain]];
```

Comment:
error  is autoreleased object and in dealloc it is released again, it will cause crash. No need of retaining error_ object. It should be -
 error = [[NSString alloc] initWithFormat:@"⚠️ %@", error_];

Reply:
Copy paste error :-) Fixed!